readme.md

```diff
@@ -16,6 +16,7 @@ Development is focused on the next major version in the `master` branch. No more
 
   - Laurent Gomila - main developer (laurent@sfml-dev.org)
   - Lukas Dürrenberger - active maintainer (eXpl0it3r@sfml-dev.org)
+  - Chris Thrasher - active maintainer (thrasher@sfml-dev.org)
```

Comment:
I'd just remove the "titles", but up to you

Reply:
I wanted to do the same